app/containers/ITunes/selectors.js

```diff
@@ -0,0 +1,19 @@
+import { createSelector } from 'reselect';
+import { initialState } from './reducer';
+import get from 'lodash/get';
+
+/**
+ * Direct selector to the iTunes state domain
+ */
+
+const selectITunesDomain = (state) => state.iTunes || initialState;
+
+export const selectITunes = () => createSelector(selectITunesDomain, (substate) => substate);
+
+export const selectSongData = () => createSelector(selectITunesDomain, (substate) => get(substate, 'songData', null));
```

Comment:
don't use the third parameter for the lodash.get here. We don't need to supply it a default value as null. 

```
createSelector(selectITunesDomain, (substate) => get(substate, 'songData'));
```

please replicate for all selectors